install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1022,15 +1022,46 @@ endif::openshift-origin[]
 ifdef::openshift-enterprise[]
 .. Follow the first step in
 xref:../../install_config/aggregate_logging.adoc#deploying-the-efk-stack[Deploying
-the EFK Stack]. After the deployer completes, as a cluster administrator,
-re-attach the persistent volumes you were previously using, then deploy a
+the EFK Stack]. After the deployer completes,
+re-attach the persistent volume claims you were previously using, then deploy a
 template that is created by the deployer:
 +
 ====
 ----
 $ oc process logging-support-template | oc apply -f -
 ----
 ====
+
+. Deployment of logging components is intended to happen automatically
+based on tags being imported into the ImageStreams created in the previous
+step. However, as not all tags are automatically imported, this mechanism
+has become unreliable as multiple versions are released. Therefore,
+manual importing may be necessary as follows.
++
+For each ImageStream `logging-auth-proxy`, `logging-kibana`,
+`logging-elasticsearch`, and `logging-fluentd`, manually import the
+tag corresponding to the `*IMAGE_VERSION*` specified (or defaulted)
+for the deployer.
++
+====
+----
+$ oc import-image <name>:<version> --from <prefix><name>:<tag>
+----
+
+For example:
+
+----
+$ oc import-image logging-auth-proxy:3.2.1 \
+     --from registry.access.redhat.com/openshift3/logging-auth-proxy:3.2.1
+$ oc import-image logging-kibana:3.2.1 \
+     --from registry.access.redhat.com/openshift3/logging-kibana:3.2.1
+$ oc import-image logging-elasticsearch:3.2.1 \
+     --from registry.access.redhat.com/openshift3/logging-elasticsearch:3.2.1
+$ oc import-image logging-fluentd:3.2.1 \
+     --from registry.access.redhat.com/openshift3/logging-fluentd:3.2.1
+----
+====
```

Comment:
You can drop the `====` block around these bits. Also the one above that was already in here (1029 - 1033), if you don't mind. These were the only instances of example blocks around code within numbered steps; it gets too busy on the docs.openshift build, and doesn't show up differently anyway on the Portal build.

Just make sure the `+`s are still good for the step after.